dev/format.sh

```diff
@@ -14,3 +14,9 @@ find src/proto/flwr/proto -name *.proto | grep "\.proto" | xargs clang-format -i
 # Examples
 python -m black -q examples
 python -m docformatter -i -r examples
+
+
+# Notebooks
+KEYS="metadata.celltoolbar metadata.language_info metadata.toc metadata.notify_time metadata.varInspector metadata.accelerator metadata.vscode cell.metadata.id cell.metadata.heading_collapsed cell.metadata.hidden cell.metadata.code_folding cell.metadata.tags cell.metadata.init_cell"
+python -m nbstripout doc/source/tutorial/*.ipynb --extra-keys "$KEYS"
+python -m nbstripout examples/*/*.ipynb --extra-keys "$KEYS"
```

Comment:
Can we add these in "check-only mode" to `test.sh`?

Reply:
@danieljanes Well I looked into it, there is an option to just check if the files would need formatting but I don't think there is an easy way to halt the execution of the script if so.